src/solve.jl

```diff
@@ -94,7 +94,12 @@ function DiffEqBase.__init(
 
     # define absolute tolerance for fixed-point iterations
     if alg.fixedpoint_abstol === nothing
-        fixedpoint_abstol_internal = integrator.opts.abstol
+        abstol = integrator.opts.abstol
+        if typeof(abstol) <: Number
+            fixedpoint_abstol_internal = abstol
+        else
```

Comment:
I'm not sure whether this specialization is required. To me it seems `recursivecopy(::Number)` falls back to `deepcopy` which both give a more complex output with e.g. `@code_native recursivecopy(3)` and `@code_native deepcopy(3)` than `@code_native copy(3)`? Should we just define `recursivecopy(x::Number) = copy(x)` in RecursiveArrayTools which by default just returns `x` (https://github.com/JuliaLang/julia/blob/master/base/number.jl#L88)?

Reply:
Yes. There shouldn't be a need to `deepcopy` a number anymore. This could improve performance in some places as well.

Reply:
We should make sure Measurements.jl still works after changing to `copy`. It should.